scripts/gulp/tasks/screener.ts

```diff
@@ -48,7 +48,11 @@ task('screener:runner', cb => {
     affectedPackages = getAffectedPackages(previousMasterCommit);
   }
 
-  if (!affectedPackages.has(docsPackageName) && isPrBuild) {
+  // debugging
+  console.log('affected package tree');
+  console.log(Array.from(affectedPackages.values()));
```

Comment:
can we wrap this into a function ? 
- we should not console.log like that
- if we really need to console.log lets make it more DRY

Reply:
yup sounds good